packages/widget-chat/src/actions/user.js

```diff
@@ -37,7 +37,7 @@ export function fetchCurrentUser(spark) {
 export function fetchUser(userEmail, spark) {
   return (dispatch) => {
     dispatch(requestUser(userEmail));
-    spark.user.getByEmail({email: userEmail})
+    spark.user.getUUID(userEmail)
       .then((user) => dispatch(receiveUser(userEmail, user)));
```

Comment:
We are still receiving the user. I just changed the method name that we are calling. After discussion with @ianwremmel, it seems like we don't need to fetch user details of "other" users since those details are returned in the conversation object. I could remove all of "non current" user requests from here. What do you think?